app.rb

```diff
@@ -1,9 +1,29 @@
 require 'sinatra/base'
+require_relative './lib/models/user'
+require_relative './lib/models/post'
 
 class Chitter < Sinatra::Base
+  # setup sinatra
+  set :root,  File.dirname(__dir__)
+  set :views, proc { File.join(__dir__, 'lib/views') }
+  set :public_folder, proc { File.join(__dir__, 'lib/assets') }
+
+  enable :sessions
+
   get '/test' do
     'Test page'
   end
 
```

Comment:
This is great but there are some unconventional names and it's worth reflecting on that.  I'd expect the route for getting all Posts to be `GET '/posts'`. You can potentially use the same route for getting the filtered list, if you add a little bit of logic to the controller action.

I'd expect the method for getting all posts to be `Post.all`